### PR TITLE
tethex: add a new package

### PR DESCRIPTION
**Pull request by davydden**

### CODE REVIEW

File: var/spack/repos/builtin/packages/tethex/package.py [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+#
+# You should have received a copy of the GNU Lesser General Public
+# License along with this program; if not, write to the Free Software
+# Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA 02111-1307 USA
+##############################################################################
+from spack import *
+
+
+class Tethex(Package):
+    """Tetxeh is designed to convert triangular (in 2D) or tetrahedral (in 3D)
```
Comment (tgamblin) [inline var/spack/repos/builtin/packages/tethex/package.py]: "Tetxeh"

File: var/spack/repos/builtin/packages/tethex/package.py [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+    Gmhs's mesh to quadrilateral or hexahedral one respectively. These meshes
```
Comment (tgamblin) [inline var/spack/repos/builtin/packages/tethex/package.py]: what's a gmhs?

File: var/spack/repos/builtin/packages/tethex/package.py [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+    homepage = "https://github.com/martemyev/tethex"
+    url      = "https://github.com/martemyev/tethex/archive/v0.0.7.tar.gz"
+
+    version('0.0.7', '6c9e4a18a6637deb4400c6d77ec03184')
+    version('develop', git='https://github.com/martemyev/tethex.git')
+
+    depends_on('cmake', type='build')
+
+    def install(self, spec, prefix):
+        # options = ['-TESTING=ON']  # TODO: need gtest (Google Test)
```
Comment (tgamblin) [inline var/spack/repos/builtin/packages/tethex/package.py]: There's a `googletest` package

File: var/spack/repos/builtin/packages/tethex/package.py (hunk 1 above)
Comment (davydden) [inline var/spack/repos/builtin/packages/tethex/package.py]: done.

File: var/spack/repos/builtin/packages/tethex/package.py (hunk 2 above)
Comment (davydden) [inline var/spack/repos/builtin/packages/tethex/package.py]: done.

File: var/spack/repos/builtin/packages/tethex/package.py (hunk 3 above)
Comment (davydden) [inline var/spack/repos/builtin/packages/tethex/package.py]: I now remove this as it looks like it is not a set of tests before installation, but something else. I also had problems enabling this feature.

**Pull request merged**